techniques/fileDistribution/fileTemplate/1.0/fileTemplate.st

```diff
@@ -180,13 +194,13 @@ pass3::
       #Case with execution
       
       "any" usebundle => rudder_common_reports_generic("fileTemplate", "command_execution_${posthook_canon[${index}]}", "${trackingkey[${index}]}", "Posthook", "${dst[${index}]}", "The command ${posthook[${index}]} from postHook execution "),
-      ifvarclass => "posthook_launch_${index}";
+      ifvarclass => "command_execution_${posthook_canon[${index}]}_reached";
 
 
       #Case without execution
 
       "report_${index}" usebundle  => rudder_common_report("fileTemplate", "result_na", "${trackingkey[${index}]}", "Posthook", "${dst[${index}]}", "No post-modification needed to run"),
-      ifvarclass => "!file_modified_${index}.posthook_specified_${index}";
+      ifvarclass => "!file_modified_${index}.posthook_specified_${index}.!command_execution_${posthook_canon[${index}]}_reached";
```

Comment:
Sorry, I don't understand: if we erase this conditional state, persistent posthook in error will report as a N/A and not as a posthook execution.